tests/test_packages/test_skills/test_carpark.py

```diff
@@ -61,6 +62,7 @@ def test_carpark(self, pytestconfig):
 
         self.add_item("connection", "fetchai/oef:0.2.0")
         self.add_item("skill", "fetchai/carpark_client:0.1.0")
+        self.set_config("agent.default_connection", "fetchai/oef:0.2.0")
```

Comment:
important to set correct default connection otherwise envelopes are improperly routed